src/config/general.rs

```diff
@@ -73,4 +87,11 @@ impl General {
             Seed512(seed)
         }
     }
+
+    pub fn validate(&self) {
```

Comment:
We might want to check that it's not less than maybe 10ms ?

Reply:
Ack, that makes sense.